packages/addons/addon-depends/tvh-dtv-scan-tables/package.mk

```diff
@@ -2,13 +2,13 @@
 # Copyright (C) 2018-present Team LibreELEC (https://libreelec.tv)
 
 PKG_NAME="tvh-dtv-scan-tables"
-PKG_VERSION="6bb0a70a0dcb97d5457c1ef4a7ccce634ea419b0"
-PKG_SHA256="a5f375d28e52ff3f527fecb7968f3ede6bc2e128d1ad78537ab4f0f7844edb45"
+PKG_VERSION="a97c9d49fa2d3e572a39c254ca5a1485ce69f376" # 2020-12-26
+PKG_SHA256="9845f0dbd1bc0c7d39bf63db684b41a0a7759a8b8ac96af1cbd0b24c4b0bb68d"
 PKG_LICENSE="GPL"
 PKG_SITE="https://github.com/tvheadend"
-PKG_URL="https://github.com/tvheadend/dtv-scan-tables/archive/$PKG_VERSION.tar.gz"
+PKG_URL="https://github.com/crazycat69/dtv-scan-tables/archive/$PKG_VERSION.tar.gz"
```

Comment:
gosh ... that meh happens if the proper upstream doesn't provide a proper way to upstream stuff that is doable for normal ppls

so we have now a wild mix between upstream https://git.linuxtv.org/dtv-scan-tables.git and those 2 forks :/

I have a think how we solve this

Reply:
Yup 😟